XLS-0065d-single-asset-vault/README.md

```diff
@@ -791,3 +921,7 @@ In addion, any compliance mechanisms applied to `USDC` will also apply to the sh
 ### A-1.4 What is the difference between the `VaultOwner` and the `pseudo-account`?
 
 XRP Ledger is an account based blockchain. That means that assets (XRP, IOU and MPT must be held by an account). The Vault Object (or any other object, such as the AMM) cannot hold assets directly. Therefore, a pseudo-account is created that holds the assets on behalf of that object. The pseudo-account is a stand-alone account, that cannot receive funds, it cannot send transactions, it is there to only hold assets. So for example, when a depositor deposits assets into a vault, in reality this transaction moves the assets from the depositor account to the pseudo-account.
```

Comment:
I think the parenthesis need to be moved, like so `... (XRP, IOU and MPT) must be held by an account.` 

Would be also good to explain in this paragraph that pseudo-account also acts as an issuer of the vault shares.